### PR TITLE
Remove unused usings

### PR DESCRIPTION
**Pull request by davkean**

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IProjectCapabilitiesService.cs [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
```
Comment (jmarolf) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IProjectCapabilitiesService.cs]: Can we remove the extra line between the header and the namespace for places where all the using are removed?

Comment (tannergooding) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IProjectCapabilitiesService.cs]: Seems like something the analyzer/quick-fix should be doing. Bug?

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IProjectCapabilitiesService.cs]: I'll file a bug.

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IProjectCapabilitiesService.cs]: https://github.com/dotnet/roslyn/issues/23568

**Comment by davkean** [comment]
Removed the blank lines and filed the bug.

**Comment by CyrusNajmabadi** [comment]
Fair enough.  I was thinking that the moment you did anything interesting you would need some namespaces.  But i can see how simple interfaces, without any code, that just use built-in primitive types could hit this.

Seems reasonable to fix then :)    Def something the community could pick up.  @sharwell Could you add the right tags?  Thanks!

**Pull request merged**